katello/katello.spec

```diff
@@ -28,15 +28,57 @@ BuildRequires: util-linux
 
 Requires: %{name}-common = %{version}-%{release}
 
+%if 0%{?rhel} == 6
+Requires: redhat-logos >= 60.0.14
+%endif
+
+Requires(post): chkconfig
+Requires(postun): initscripts coreutils sed
+Requires(pre): shadow-utils
+Requires(preun): chkconfig
+Requires(preun): initscripts
```

Comment:
i'm not sure these are actually needed anymore?  We aren't starting or stopping (or enabling) services, are we doing anything at all in %pre or %post in either rpm?